CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+Contributing Guidelines
```

Comment:
headings should be added throughout this file, it's currently hard to read

Reply:
I'll do

Reply:
I made some improvements